stock_valued_picking_report/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Stock Valued Picking Report
+===========================
+
+Add amount information to Delivery Slip report.
+You can select at partner level if picking list report must be valued or not.
```

Comment:
If the partner by default has set not to receive valued picking report, is it possible to choose this option in the stock picking?

Reply:
@rafaelbn No, it's not possible. It checks it at document level. Maybe it wouldn't be hard to have an option in the order to override partner's preference.